### ISSUE TITLE
Unable to open some vehicle sheets

### ISSUE THREAD

**Issue by McCookies** [kypvalanx/Foundry-VTT-StarWars-SagaEdition#506]
I'm unable to open the Actor sheets for the PCs ships.  They are slightly customized versions of the Citadel cruiser and Modified Z-95 headhunter.  Other vehicle sheets (all unmodified) open fine.

Console shows the following when I try to open any of the 3 sheets:

Error: An error occurred while rendering SWSEActorSheet 
25. source?.get is not a function
    at Hooks.onError (foundry.js:654:24)
    at foundry.js:5795:13
Caused by: TypeError: source?.get is not a function
    at getDocumentByUuid (util.mjs:1392:42)
    at getActor (attack.mjs:975:16)
    at get provider (attack.mjs:183:16)
    at lookupProperty (handlebars.min.js:27:24258)
    at eval (eval at createFunctionContext (handlebars.min.js:29:10011), <anonymous>:11:190)
    at Object.h [as inverse] (handlebars.min.js:27:25975)
    at Object.<anonymous> (handlebars.min.js:27:18195)
    at Object.<anonymous> (handlebars.min.js:27:18372)
    at Object.c (handlebars.min.js:27:27865)
    at eval (eval at createFunctionContext (handlebars.min.js:29:10011), <anonymous>:10:54)
<img width="597" alt="console" src="https://github.com/user-attachments/assets/3505b382-209d-4c64-a0c2-2f17f463e83b">

**Comment by McCookies** [comment]
Tested some more.  I added a YT-1300 transport from the compendium to my actors.  I was able to open the Actor sheet with no problem.  I tried to make some changes to it (crew quality, add some systems).  That went fine until I tried dragging a weapon system over to the pilot or gunner slot.  It didn't stay and the Actor sheet is not openable after that.
<img width="565" alt="Screenshot 2024-10-26 at 15 48 34" src="https://github.com/user-attachments/assets/e1f8d30f-f980-4f35-a7aa-dd364e6f8413">
